File: doc/src/codes/Users_Manual/modulation/fourier_0.cpp
#include "autd3.hpp"

auto m =
    autd3::modulation::Fourier(autd3::modulation::Sine(100))
        .add_component(autd3::modulation::Sine(150))
        .add_components_from_iter(std::vector{autd3::modulation::Sine(200)});

File: doc/src/codes/Users_Manual/modulation/fourier_0.rs
# extern crate autd3;
# use autd3::prelude::*;
use autd3::modulation::Fourier;

# #[allow(unused_variables)]
# fn main()  {
let m = Fourier::from(Sine::new(100.))
        .add_component(Sine::new(150.))
        .add_components_from_iter([Sine::new(200.)]);
# }